network-api/networkapi/wagtailpages/templates/wagtailpages/static/styleguide.html

```diff
@@ -357,6 +357,41 @@ <h3 class="h2-heading mt-5">Mini Version & Stacked</h3>
         <div class="share-button-group-wrapper" data-version="mini" data-layout="stacked"></div>
       </div>
 
+      <h2 class="h1-heading mt-5">Dropdown Field</h2>
+      <hr class="hr-emphasis">
+
+      <h3 class="h2-heading mt-5">Default</h3>
+      <hr>
+
+      <div class="mb-4">
+        <select class="dropdown-field">
+          <option value="">.dropdown-field on <code class="d-inline">&lt;select></code></option>
+        </select>
+      </div>
+
+      <div class="mb-5">
+        <select class="dropdown-field-large">
+            <option value="">.dropdown-field-large on <code class="d-inline">&lt;select></code></option>
```

Comment:
Let's move the label/note out from the `<option>` here and make the dropdown example have multiple options. That way we will be able to test selected/unselected option styles. 

For example, 

```html
<h4 class="h3-heading">Large</h3>
<select class="large">
  <option>Option 1</option>
  <option>Option 2</option>
  <option>Option 3</option>
</select>
```